src/overlays/actors/ovl_Arrow_Fire/z_arrow_fire.h

```diff
@@ -8,11 +8,18 @@ struct ArrowFire;
 typedef void (*ArrowFireActionFunc)(struct ArrowFire*, GlobalContext*);
 
 typedef struct ArrowFire {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x114];
-    /* 0x0258 */ ArrowFireActionFunc actionFunc;
-    /* 0x025C */ char unk_25C[0x8];
-} ArrowFire; // size = 0x264
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ ColliderQuad collider1;
+    /* 0x1C4 */ ColliderQuad collider2;
```

Comment:
What's the difference between the use of these two colliders?

Reply:
I'm not sure other than the sizes being different.